src/main/java/com/google/gcs/sdrs/dao/BaseDao.java

```diff
@@ -31,6 +29,11 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.google.gcs.sdrs.dao.model.PooledStsJob;
```

Comment:
google-java-format doesn't seem to be working properly. Please check it is the latest 1.7.0.1

Reply:
fixed